src/Entity/Form/DeveloperAppCreateForm.php

```diff
@@ -73,6 +73,9 @@ public static function create(ContainerInterface $container) {
 
   /**
    * {@inheritdoc}
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
```

Comment:
I am little bit sceptical whether we should document *all* possible exceptions that Phpstorm offers for us. If we know for sure that an exception should not be thrown in normal circumstances (confidence lvl 99%) then we should not document it, because it could be misleading.
Ex.: `\Drupal\Component\Plugin\Exception\PluginNotFoundException` should never be thrown because "api_product" entity always exists in our module.